README.md

```diff
@@ -185,27 +185,31 @@ or
 
 ### Build & Publish
 
-#### Web Component & Angular Wrapper - via a PR
+#### Preparing to publish the Web Component & Angular Wrapper
 
-The publication of the web component and its angular wrapper has been automated in the `release.yml` workflow. To trigger it, submit a pull request to the `release` branch and get it reviewed. Publication will happen when the PR is merged.
+Publication to npmjs is mostly automated via the tag-triggered `release.yml` CI workflow, but first you have to update the repo with the new version number in multiple places:
 
-#### Web Component & Angular Wrapper - manually
+- Update the version number for web-component and ngx-web-component in [packages/web-component/package.json](packages/web-component/package.json) and [packages/ngx-web-component/package.json](packages/ngx-web-component/package.json).
+- Nx does not manage dependencies among projects for us, so update the version in `peerDependencies` for `@readalongs/web-component` in [packages/ngx-web-component/package.json](packages/ngx-web-component/package.json).
+- Update `singleFileBundleVersion` in [packages/studio-web/package.json](packages/studio-web/package.json).
+- All the versions above should be exactly in sync!
```

Comment:
As mentioned above... we should verify this in the release workflow and fail if not.  I will try adding some code to do this shortly.